be/src/olap/primary_key_index.cpp

```diff
@@ -32,7 +32,9 @@
 #include "olap/types.h"
 
 namespace doris {
-
+PrimaryKeyIndexBuilder::~PrimaryKeyIndexBuilder() {
+        _bloom_filter_index_mem_tracker->release(_bloom_filter_index_mem_tracker->consumption());
```

Comment:
`MemTracker` does not need to release itself during destruction.
Because `MemTracker` is an independent entity and will not exist after destruction.

Of course that's okay